### DISCUSSION TITLE
Config Provider文档中提到的定制主题的文档链接404了

### DISCUSSION THREAD

**Discussion by punkin6** [ant-design/ant-design#39264] [Q&A]
在文档https://github.com/ant-design/ant-design/blob/master/components/config-provider/index.zh-CN.md中提到了定制主题的文档，如下图，这个链接是404了，新文档的链接是什么？

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/118983190/205582693-e2559e8c-fa88-48df-822f-1fa3fff48b25.png">

在https://github.com/ant-design/ant-design/blob/598b5223fdefe12fcc9f30b2fba10ae30ce9d23a/README-zh_CN.md这个文档中提到的链接（https://ant.design/docs/react/customize-theme-cn）是可以用的。是否更新一下ConfigProvider中的文档链接？

**Comment by MadCcc** [comment]
欢迎 PR～
